generators/ci-cd/index.js

```diff
@@ -185,6 +189,26 @@ module.exports = class extends BaseBlueprintGenerator {
           this.frontTestCommand = 'test';
         }
       },
+      derivedProperties() {
+        this.isBuildToolGradle = this.buildTool === GRADLE;
```

Comment:
If we are going to use this by approach by default:
- Extract to a generator-base function: `loadDerivedProperties`.
- Remove is/has from properties.
- Use directly from configuration this.jhipsterConfig.

```suggestion
        this.buildToolGradle = this.jhipsterConfig.buildTool === GRADLE;
```

Reply:
Our just implement those derived properties in load*Config.

Reply:
Didn't thought about this one :-), simpler and perfect fit!

Reply:
@mshima Better now?